pkg/functions/riff/driver_test.go

```diff
@@ -102,8 +102,7 @@ func TestDriver_Create(t *testing.T) {
 	}
 
 	err := d.Create(&f, &functions.Exec{
-		Image:    "imikushin/dispatch-riff-nodejs6-base:0.0.3-dev1",
-		Language: "nodejs6",
+		Image: "imikushin/dispatch-riff-nodejs6-base:0.0.3-dev1",
```

Comment:
We shouldn't have references to external images.  I can create a repo for you if you need.

Reply:
Do we still need this image? IIUC, we don't need FaaS-specific images anymore.